lib/common/benchmarkModelGenerator.js

```diff
@@ -0,0 +1,408 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { Blob } = require('buffer');
+const {
+    uniqueNamesGenerator, adjectives, colors, animals
+} = require('unique-names-generator');
```

Comment:
This library doesn't appear to be maintained. It has high adoption, but a low number of contributions. That gives us a high supply-chain risk.

Is there a simpler (and perhaps less pretty) way to do the same thing without a dependency?

Reply:
We could just have a string with an index concatenated at the end e.g:
- `Model0Declaration0`
- `Model0Declaration0Property0`

Would you prefer that I go down this route?

Reply:
Yes, that's sufficient for our purposes, right now. In future, I would imagine that the seed could be pluggable (e.g. psuedo-random, client-provided, fuzzed etc.)

Reply:
Thanks! Addressed ✅